src/concrete/ml/torch/hybrid_model.py

```diff
@@ -188,28 +366,36 @@ def _get_module_by_name(model: nn.Module, name: str) -> Union[RemoteModule, nn.M
         Raises:
             ValueError: If no module found for the given name.
         """
+        # TODO: Shouldn't this search recursively in name modules of name modules?
         for module_name, module in model.named_modules():
             if module_name == name:
                 return module
         raise ValueError(f"No module found for name {name}")
 
-    def init_client(self, path_to_client: str, path_to_keys: str):
+    def init_client(
+        self, path_to_clients: Optional[Path] = None, path_to_keys: Optional[Path] = None
+    ):  # pragma:no cover
         """Initialize client for all remote modules.
 
         Args:
-            path_to_client (str): Path to the client.zip files.
-            path_to_keys (str): Path to the keys folder.
+            path_to_clients (Optional[Path]): Path to the client.zip files.
+            path_to_keys (Optional[Path]): Path to the keys folder.
         """
-        # TODO: https://github.com/zama-ai/concrete-ml-internal/issues/3869
-        # implement client initialization
+        if path_to_clients is None:
+            path_to_clients = Path("clients")
+        path_to_clients.mkdir(exist_ok=True)
+        for module_name, module in self.remote_modules.items():
+            path_to_client = path_to_clients / module_name
+            path_to_client.mkdir(exist_ok=True)
+            module.init_fhe_client(path_to_client=path_to_client, path_to_keys=path_to_keys)
 
     def compile_model(
         self,
         x: torch.Tensor,
         n_bits: int = 8,
-        rounding_threshold_bits: int = 8,
-        p_error=0.01,
-        configuration: Configuration = None,
+        rounding_threshold_bits: Optional[int] = 8,
+        p_error: float = 0.01,
```

Comment:
Default should follow the one from CML

Reply:
alright yeah I can change that, it was like that before so I didn't consider changing it

Reply:
Changed it back to defaults

Reply:
thanks